docs/technology_stack.adoc

```diff
@@ -7,35 +7,45 @@
 toc::[]
 
 == Overview
-The following sections describe the technology stack used by Planner.
+The *Planner* is an *Angular* app built on top of *Node.js*. It can run as a standalone app, or be used as a library to be integrated into another app, like link:./fabric8-ui[Fabric8 UI].
 
-== Angular 4
-<TBD>
+Development or deployment of *Planner* is fairly platform agnostic, so we can skip the operating system requirements and discuss from the runtime environment onward.
 
-== CSS and SASS
-<WIP, to be changed to PF CSS, Less>
+== Development Runtime
+Planner uses *`Node.js`* as the platform-runtime, on top of which it builds; and *`npm`* for resolving and distributing packages for the project.
 
-Fabric8-planner uses SASS for it's stylesheets. It also uses the Angular emulation
-of the shadow dom, hence it is recommended to place your styles in the
-*_.component.scss_* file next to the HTML and the TypeScript.
+We keep ourselves compatible with the latest stable releases of *`Node.js`* & *`npm`* - however, *`Node.js >= 6.xx`* & *`npm >= 3.xx`* releases are recommended.
 
-If you want to create a shared style that could be used by multiple components, then we recommend adding it as a mixin to
-*_src/assets/stylesheets/_planner-mixins.scss_*. The mixins are imported into every
-*_.component.scss_* file.
+== Development Pipeline
 
-You can then create a real class by doing something like:
+Following are the components or parts of the process followed in order to develop, build, test and deploy *Planner*. Most of these components are standard, with limited project specific customizations and configurations applied.
 
-    .my-class {
-      @include my-class;
-    }
+Task Runner::
+We use *Gulp* for running predefined tasks. These tasks include, compiling or preprocessing the LESS/TypeScript sources into CSS/JS, building the distributable artifacts, running dev-server in watch mode and general house-keeping (install, reinstall, cleanup etc).
```

Comment:
It would be nice to use a bulleted list here.
These tasks include:
- compiling or preprocessing the LESS/TypeScript sources into CSS/JS 
- building the distributable artifacts
- running dev-server in watch mode 
- general house-keeping (install, reinstall, cleanup etc)